lib/log_parser.rb

```diff
@@ -9,4 +11,16 @@ def initialize(file)
   def read_first_line
       File.readlines(@file).first
   end
+
+  def lines_counter
+    File.readlines(@file).count
+  end
+
+  def json_output
+    data = {
```

Comment:
Variavel data nao usada

```suggestion
    {
```